ynput/OpenPype
# Code: Tweak docstrings and return type hints

## Changelog Description

Tweak docstrings and return type hints for functions in `openpype.client.entities`.

## Additional info

As [reported on discord](https://discord.com/channels/517362899170230292/517382145552154634/1098511261291003966) this intends to solve the issue I had:

> I'm having some issues with many functions in the `openpype.client.entities` package. My IDE keeps telling me functions like these return `None` instead of `dict` or `None.` For example for `get_representation_by_name` https://github.com/ynput/OpenPype/blob/9aa8aa469fc3a81e714207809af786b520043bf6/openpype/client/entities.py#L1004-L1020

## Testing notes:

1. Launch your IDE with the codebase and hope it likes it.
2. Make sure you can read the docstrings and the are correct.
